<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Editable Tree Model</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QTreeView" name="view">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>175</width>
         <height>661</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>175</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="animated">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="DrawBoardArea" name="widget_2" native="true">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>0</y>
         <width>821</width>
         <height>661</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>0</y>
         <width>20</width>
         <height>631</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="actionsMenu">
    <property name="title">
     <string>&amp;Actions</string>
    </property>
    <addaction name="insertRowAction"/>
    <addaction name="insertColumnAction"/>
    <addaction name="separator"/>
    <addaction name="removeRowAction"/>
    <addaction name="removeColumnAction"/>
    <addaction name="separator"/>
    <addaction name="insertChildAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="actionsMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="exitAction">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="insertRowAction">
   <property name="text">
    <string>Insert Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, R</string>
   </property>
  </action>
  <action name="removeRowAction">
   <property name="text">
    <string>Remove Row</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, R</string>
   </property>
  </action>
  <action name="insertColumnAction">
   <property name="text">
    <string>Insert Column</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I, C</string>
   </property>
  </action>
  <action name="removeColumnAction">
   <property name="text">
    <string>Remove Column</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, C</string>
   </property>
  </action>
  <action name="insertChildAction">
   <property name="text">
    <string>Insert Child</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DrawBoardArea</class>
   <extends>QWidget</extends>
   <header location="global">drawboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
